Controllers/WordsController.cs

```diff
@@ -89,25 +63,5 @@ public async Task<IActionResult> PostWord(Word word)
 
             return Ok(word);
         }
-
-        [HttpDelete("{id}")]
-        public async Task<IActionResult> DeleteWord(Guid id)
-        {
-            var word = await _context.Words.FindAsync(id);
-            if (word == null)
-            {
-                return NotFound();
-            }
-
-            _context.Words.Remove(word);
-            await _context.SaveChangesAsync();
-
-            return NoContent();
-        }
-
-        private bool WordExists(Guid id)
-        {
-            return _context.Words.Any(e => e.Id == id);
-        }
     }
 }
```

Comment:
Add a new line at the end of all files. There is a problem in Unix systems with it, you can google about it for more information.